### PR TITLE
Fix sporadic ResizeObserver loop limit exceeded error

### PR DESCRIPTION
**Pull request by whather**

### CODE REVIEW

File: src/with-content-rect.js [hunk 1]
```diff
@@ -63,12 +63,11 @@ function withContentRect(types) {
         this._animationFrameID = window.requestAnimationFrame(() => {
           if (this._resizeObserver !== null) {
             this.setState({ contentRect })
+            if (typeof this.props.onResize === 'function') {
+              this.props.onResize(contentRect)
```
Comment (whather) [inline src/with-content-rect.js:67]: This fixed the issue locally for me and seems to be what other solutions have done as well: https://github.com/maslianok/react-resize-detector/commit/9625f67a48ee76a788b316c3fede083a419bd661#diff-1fdf421c05c1140f6d71444ea2b27638R82

Comment (whather) [reply src/with-content-rect.js:67]: @souporserious lemme know if this makes sense. Thanks!

Comment (souporserious) [reply src/with-content-rect.js:67]: Makes sense! Thanks for the PR! 🙏

**Comment by souporserious** [comment]
Thanks for the PR! Released under `2.2.5` 🎉

**Pull request merged**